install_dependencies.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+# Direct Package Dependencies
+DEPENDENCIES="libproj-dev"
+# Library Dependencies
+LIBRARY_DEPENDENCIES=""
```

Comment:
What is the difference between `DEPENDENCIES` and `LIBRARY_DEPENDENCIES`?

Reply:
Not sure what Robs initial idea here was. The only thing that makes sense to me is one to capture direct dependencies and indirect dependencies (dependencies of dependencies). The only benefit I could see in separating them is maybe maintenance. 
It may be easier to ensure all dependencies included are necessary if there is a distinction between direct and indirect dependencies.
I am also willing to remove this since I can really think of strong justification for keeping this. I'm indifferent either way. I'll let you make the call.

Reply:
Let's keep it for now. I'm not knowledgeable enough right now to form an opinion on it.